### DISCUSSION TITLE
Missing artifact org.eclipse.rdf4j:spring-boot-sparql-web:jar:4.0.2

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#4003] [Q&A] [answered]
Hi, i try to setup a simple spring-boot sample but cannot find the artifact.
i was expecting it also [here](https://mvnrepository.com/artifact/org.eclipse.rdf4j/spring-boot-sparql-web)

Thanks Fredy

**Comment by naturzukunft** [comment]
The artifactId in the [doc](https://github.com/eclipse/rdf4j/tree/main/spring-components/spring-boot-sparql-web#readme) seems to be wrong!
Should be `rdf4j-spring-boot-sparql-web` ?!

**Comment by naturzukunft** [comment]
Another typo in the Readme is `retun sailRepository;`

**Reply by JervenBolleman** [reply]
Thanks I will fix both issues.

**Comment by JervenBolleman** [comment, marked as answer]
These are two issues and I opened a pull request. https://github.com/eclipse/rdf4j/pull/4006 I hope this fixes the basic issue. But I would be happy to hear about anything else I could improve.
